pkg/api/frontendsettings.go

```diff
@@ -25,6 +25,53 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
+// Returns a file that is easy to check for changes
+// Any changes to the file means we should refresh the frontend
```

Comment:
how do we make sure it doesn't result in a loop if there are issues? Can we limit the amount of refreshes?